### PR TITLE
feat(umd): inject css files

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
CSS files coming from the project are not injected by the initiator script

**What is the chosen solution to this problem?**
Add them to the body

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: cdn/webpack-inject/src/inject.js [hunk 1]
```diff
@@ -1,112 +1,122 @@
...
-		document.body.appendChild(style);
-	});
-}
+	function addStyles(urls) {
+		urls.forEach(function onUrl(url) {
+			// @see https://stackoverflow.com/questions/32759272/how-to-load-css-asynchronously
+			const style = document.createElement('link');
+			style.setAttribute('rel', 'stylesheet');
+			style.setAttribute('media', 'print');
+			style.setAttribute('onload', 'this.media="all"');
```
Comment (jmfrancois) [inline cdn/webpack-inject/src/inject.js:43]: great !

File: cdn/webpack-inject/src/inject.js [hunk 2]
```diff
@@ -1,112 +1,122 @@
...
-		if (TUI.indexOf(info.name) !== -1 && window.TALEND_CDN_VERSIONS.TUI !== '@@TALEND_UI_VERSION@@') {
-			return window.TALEND_CDN_VERSIONS.TUI;
+	function getProjectVersion(info) {
+		if (TUI.indexOf(info.name) !== -1 && talendCdnVersions.TUI !== '@@TALEND_UI_VERSION@@') {
+			return talendCdnVersions.TUI;
 		}
-		return window.TALEND_CDN_VERSIONS[info.name];
+		return talendCdnVersions[info.name];
 	}
-	return undefined;
```
Comment (jmfrancois) [inline cdn/webpack-inject/src/inject.js:77]: no lint error ?

Comment (frassinier) [reply cdn/webpack-inject/src/inject.js:77]: no 🤔

**Review by jmfrancois — APPROVED** [review]

File: cdn/webpack-inject/src/inject.js (hunk 2 above)
Comment (jmfrancois) [reply cdn/webpack-inject/src/inject.js:77]: my bad I missed the remove of on if level

**Pull request merged**